public/app/features/dashboard/components/ShareModal/ShareExport.tsx

```diff
@@ -137,9 +138,11 @@ export class ShareExport extends PureComponent<Props, State> {
         <Field label="Export for sharing externally">
           <Switch value={shareExternally} onChange={this.onShareExternallyChange} />
         </Field>
-        <Field label="Export with default values removed">
-          <Switch value={trimDefaults} onChange={this.onTrimDefaultsChange} />
-        </Field>
+        {config.featureToggles.trimDefaults && (
+          <Field label="Export with trimed dashboard json">
```

Comment:
This should probably still be 

```suggestion
          <Field label="Export with default values removed">
```

Reply:
Plus one